lib/Github/HttpClient/CachedHttpClient.php

```diff
@@ -27,6 +27,13 @@ class CachedHttpClient extends HttpClient
     private $lastCachedResponse;
 
     /**
+     * $path + query parameter(s) if they exist.
+     *
+     * @var string
+     */
+    private $path;
```

Comment:
@stof "please don't store this in a property."
I added it in a property because it is called in two methods ! `request()` and `createRequest()`. What do you suggest so?

"and btw, the naming is wrong: the value stored in this property is not the path"
`$path` is implemented everywhere and in other places it is called `$id`...
What do you suggest so? How do you want to call $path + query?